rest/taskana-rest-spring/src/main/java/pro/taskana/rest/TaskController.java

```diff
@@ -90,7 +90,7 @@
     @GetMapping
     @Transactional(readOnly = true, rollbackFor = Exception.class)
     public ResponseEntity<PagedResources<TaskSummaryResource>> getTasks(
-        @RequestParam MultiValueMap<String, String> params) throws InvalidArgumentException {
+            @RequestParam MultiValueMap<String, String> params) throws InvalidArgumentException {
```

Comment:
We need to use the same linting I can't identify what do you have changed in rest part.

Reply:
yes, nothing should have changed here, this actually should be the master version.